### ISSUE TITLE
List Azure resource groups

### ISSUE THREAD

**Issue by micheelengronne** [mondoohq/cnquery#772]
I would like to ensure naming conventions on all my Azure resources, included resource groups.

**Describe the solution you'd like**
I would like to be able to parse a resource group as any other resource.

**Describe alternatives you've considered**
Custom azure-cli commands.

**Additional context**
Azure resource groups do not appear as specific resource in cnquery. They are not queriable and when empty they are not even listed.

**Comment by chris-rock** [comment]
Thank you @micheelengronne This is a great use case.

**Comment by chris-rock** [comment]
Thank you @micheelengronne This will be included in 7.13 which we release today. Then you have the full object available:

```
cnquery> azure.resourceGroups
azure.resourceGroups: [
  0: azure.resourcegroup name="demo_group" location="centralus"
  1: azure.resourcegroup name="test_group" location="centralus"
  2: azure.resourcegroup name="demo" location="eastus"
  3: azure.resourcegroup name="ubuntu_group" location="westus"
  4: azure.resourcegroup name="chris-dev_group" location="westus"
]
```

**Comment by micheelengronne** [comment]
Thank you very much.
